v4/export/ir_impl.go

```diff
@@ -193,13 +193,14 @@ func splitTableDataIntoChunks(
 		return
 	}
 
-	var max uint64
-	var min uint64
-	if max, err = strconv.ParseUint(smax.String, 10, 64); err != nil {
+	max := new(big.Int)
+	min := new(big.Int)
+	var ok bool
+	if max, ok = max.SetString(smax.String, 10); !ok {
 		errCh <- errors.WithMessagef(err, "fail to convert max value %s in query %s", smax.String, query)
```

Comment:
There is no `err` here.
```suggestion
		errCh <- errors.WithMessagef(err, "fail to convert max value %s in query %s", smax.String, query)
```